dubbo-common/src/main/java/org/apache/dubbo/common/utils/TimeUtils.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dubbo.common.utils;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Provide currentTimeMillis acquisition for high-frequency access scenarios.
+ */
+public final class TimeUtils {
+
+    private static volatile long currentTimeMillis;
+
+    private static volatile boolean isTickerAlive = true;
+
+    static {
+        currentTimeMillis = System.currentTimeMillis();
+        Thread ticker = new Thread(() -> {
+            while (isTickerAlive) {
+                currentTimeMillis = System.currentTimeMillis();
+                try {
+                    TimeUnit.MILLISECONDS.sleep(1);
+                } catch (InterruptedException e) {
+                    isTickerAlive = false;
+                    Thread.currentThread().interrupt();
+                } catch (Exception ignored) {
+                    //
+                }
+            }
+        });
+        ticker.setDaemon(true);
+        ticker.setName("time-millis-ticker-thread");
+        ticker.start();
+        Runtime.getRuntime().addShutdownHook(new Thread(ticker::interrupt));
+    }
```

Comment:
1. Prevent add complex logic in the process of class initialization
2. Start on demand

Reply:
> 1. Prevent add complex logic in the process of class initialization
> 2. Start on demand

Fixed.